[2024-05-30T03:11:40.186+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: conference_dag.set_up_database manual__2024-05-30T03:11:12.953428+00:00 [queued]>
[2024-05-30T03:11:40.197+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: conference_dag.set_up_database manual__2024-05-30T03:11:12.953428+00:00 [queued]>
[2024-05-30T03:11:40.197+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-05-30T03:11:40.198+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-05-30T03:11:40.198+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-05-30T03:11:40.212+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): set_up_database> on 2024-05-30 03:11:12.953428+00:00
[2024-05-30T03:11:40.218+0000] {standard_task_runner.py:55} INFO - Started process 3538 to run task
[2024-05-30T03:11:40.220+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'conference_dag', 'set_up_database', 'manual__2024-05-30T03:11:12.953428+00:00', '--job-id', '389', '--raw', '--subdir', 'DAGS_FOLDER/conference_dag.py', '--cfg-path', '/tmp/tmpekx3qq1p']
[2024-05-30T03:11:40.222+0000] {standard_task_runner.py:83} INFO - Job 389: Subtask set_up_database
[2024-05-30T03:11:40.289+0000] {task_command.py:388} INFO - Running <TaskInstance: conference_dag.set_up_database manual__2024-05-30T03:11:12.953428+00:00 [running]> on host 731e27f62632
[2024-05-30T03:11:40.347+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=conference_dag
AIRFLOW_CTX_TASK_ID=set_up_database
AIRFLOW_CTX_EXECUTION_DATE=2024-05-30T03:11:12.953428+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-05-30T03:11:12.953428+00:00
[2024-05-30T03:11:40.349+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-05-30T03:11:40.351+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'set -e; python /opt/***/mysql_utils/set_up_database.py']
[2024-05-30T03:11:40.366+0000] {subprocess.py:86} INFO - Output:
[2024-05-30T03:11:40.725+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-30T03:11:40.746+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=conference_dag, task_id=set_up_database, execution_date=20240530T031112, start_date=20240530T031140, end_date=20240530T031140
[2024-05-30T03:11:40.795+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-05-30T03:11:40.820+0000] {taskinstance.py:2578} INFO - 3 downstream tasks scheduled from follow-on schedule check
